packages/nimble-components/package.json

```diff
@@ -127,7 +127,7 @@
     "prettier-eslint": "^16.3.0",
```

Comment:
Looks like this has some build issues which need investigation. I can add it to my list or someone else is welcome to investigate.

 > @ni/nimble-components@32.13.0 build-components
> tsc -p ./tsconfig.json

```
Error: src/table-column/base/tests/table-column.spec.ts(89,41): error TS2345: Argument of type 'Spy<(error: Error) => void>' is not assignable to parameter of type 'Error'.
  Property 'message' is missing in type 'Spy<(error: Error) => void>' but required in type 'Error'.
```

https://github.com/ni/nimble/actions/runs/13605875322/job/38037278796?pr=2547

Reply:
I fixed this by removing the function that was doing a cast to workaround the issue that @rajsite had reported here.
https://github.com/DefinitelyTyped/DefinitelyTyped/issues/61819#issuecomment-1502152632

I guess that has been fixed in the version of `@types/jasmine` that this PR is uptaking.